### PR TITLE
fix k8s nightly test

### PR DESCRIPTION
**Pull request by Eslam-Nawara**
### Description

- k8s nightly tests failed due to failure in getting the number of ready nodes
- it failed as it was using the worker ygg_ip to get the ready nodes, the problem occurred due to the master and woker containing ygg_ip string, so when searched the output to find ygg_ip, the worker_ygg_ip was retrived and used as the ip of the master

### Changes

renamed the master ygg ip to mr_ygg_ip

### Related Issues

- #794

### CODE REVIEW

**Comment by Eslam-Nawara** [comment]
- The problem now is that the scheduler doesn't return any nodes for some reason, reported in [this](https://github.com/threefoldtech/terraform-provider-grid/issues/806) issue

- locally I tested the update using by removing the scheduler and using nodes 13,14 and the test passed 

image

File: integrationtests/k8s_test.go [hunk 1]
```diff
@@ -74,6 +79,8 @@ func TestK8s(t *testing.T) {
 		ok = TestConnection(workerIP, "22")
 		assert.True(t, ok)
 
+		time.Sleep(10 * time.Second)
```
Comment (rawdaGastan) [inline integrationtests/k8s_test.go:82]: why you sleep here?

Comment (Eslam-Nawara) [reply integrationtests/k8s_test.go:82]: For some reason the command fails if I don't wait enough, fa I added sleep, not sure if the problem is connection in my side or what

Comment (rawdaGastan) [reply integrationtests/k8s_test.go:82]: add it in the assert nodes function itself

**Review by rawdaGastan — APPROVED** [review]

**Pull request merged**